website/docs/reference/integrations/jira-cloud-plugin-installation.md

```diff
@@ -31,7 +31,11 @@ You will need a Jira admin user.
 
 ## Installation
 
-If you have an Unleash enterprise license you're welcome to reach out to us at sales@getunleash.io for a free plugin license, otherwise you'll need to try the plugin for 30 days free or purchase a license through the marketplace.
+Start by visiting the [Landing page for the Unleash Enterprise For Jira plugin on Atlassian marketplace](https://marketplace.atlassian.com/apps/1231447/unleash-enterprise-for-jira?hosting=cloud&tab=overview).
+
+![Atlassian marketplace landing page for the Unleash Enterprise For Jira plugin.](/img/unleash-marketplace-for-unleash-enterprise-jira-plugin.png)
+
+Click the yellow button in the top right corner labeled Get it now, choose which site to install the app and click the blue button Install app located at the bottom right.
```

Comment:
This is a bit tough for visually impaired folks to follow, we're generally pretty good in our docs about that. Not sure if we have a way to handle this here

Reply:
Ha! Thomas has already given you some suggestions, ignore me

Reply:
Yeah, I saw that, but we do provide the labels for the buttons as well. So as far as I know, that should cover it. I don't know what else we could do. Have any other suggestions?

Reply:
Yeah. The screenshot doesn't provide much extra detail. I think the alt text is fine too. From my (admittedly limited) knowledge, I think we've gotten the best of both worlds here: both described the label for visually impaired folks and described the location and color for those who benefit from that. So kudos to @daveleek for getting them both in 🙌🏼